__tests__/index.test.js

```diff
@@ -83,4 +83,27 @@ describe('/api/articles/:article_id', () => {
         expect(response.body.msg).toBe('Bad request');
       });
   });
+})
+
+describe('/api/articles', () => {
+  test('GET:200 Responds with an articles array of article objects', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((response) => {
+          expect(response.body.articles.length).toBe(13);
+          expect(response.body.articles).toBeSortedBy('created_at', {descending: true });
+        response.body.articles.forEach((article) => {
+          expect(typeof article.article_id).toBe('number');
+          expect(typeof article.title).toBe('string');
+          expect(typeof article.topic).toBe('string');
+          expect(typeof article.author).toBe('string');
+          expect(typeof article.created_at).toBe('string');
+          expect(typeof article.votes).toBe('number');
+          expect(typeof article.article_img_url).toBe('string'); 
+          expect(typeof parseInt(article.comment_count)).toBe('number');
+          expect(article).not.toHaveProperty('body'); 
+        });
```

Comment:
Really good test. I would recommend looking into .toMatchObject as a nice alternative.

Reply:
Thank you. I have implemented .toMatchObject. This change will be part of the commit for Task 7.